README.md

```diff
@@ -0,0 +1,206 @@
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise, you can also remove it. -->
+  <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQotruOGm3oh1D6p0rArEcdTS5DDvsvIPeegQ&usqp=CAU" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Budget App</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [👩‍💻 Presentation](#Presentation)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 ExpenseTracker <a name="about-project"></a>
+
+**[Budget App]** ExpenseTracker is a fully functioning website that allows a user to fully manage his/her expenses by keeping track of all payments and their related categories. For each category the user can see how much is already spent. A user can add categories and payments related to one or more categories.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Rails</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>PostgreSQL</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[A login page]**
+- **[Adding categories and creating expense]**
+- **[A categories list view]**
+- **[Use validations for models]**
+- **[Write unit tests for models]**
+- **[Write request tests for models]**
+- **[Set up the view model]**
+- **[Use preprocessed HTML file with embedded Ruby code.]**
+- **[Use layouts and templates for shared content.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+[Live Demo](https://expense-tracker-814r.onrender.com)
```

Comment:
- [ ] Please work around your live demo issues to fix them, I cannot sign up or login because it reaches a dead end on the deployed site. 🙏